### ISSUE TITLE
Agents walking through walls/lava?

### ISSUE THREAD

**Issue by danieljarne** [epignatelli/navix#82]
Hi!

I have been training ppo agents in navix environments and I keep seeing that agents walk through walls/lava. I'm using version 0.6.14 of Navix. I'm using a PPO agent from PureJaxRL (purejaxrl/ppo_minigrid.py). I train the agent, and then load a network to render the behaviour, and I get the following:
```python
from ppo_minigrid import *
from navix import observations, rewards, terminations, register_env
import navix

config = {
    "LR": 2.5e-4,
    "NUM_ENVS": 16,
    "NUM_STEPS": 256,
    "TOTAL_TIMESTEPS": 2e6,
    "UPDATE_EPOCHS": 1,
    "NUM_MINIBATCHES": 8,
    "GAMMA": 0.99,
    "GAE_LAMBDA": 0.95,
    "CLIP_EPS": 0.2,
    "ENT_COEF": 0.05,
    "VF_COEF": 0.5,
    "MAX_GRAD_NORM": 0.5,
    "ACTIVATION": "tanh",
    "ENV_NAME": "Navix-LavaGapS7-v0",
    "ANNEAL_LR": True,
    "DEBUG": True,
}

rng = jax.random.PRNGKey(30)
train_jit = jax.jit(make_train(config))
out = train_jit(rng)

import matplotlib.pyplot as plt
import jax
import jax.numpy as jnp

env = NavixGymnaxWrapper("Navix-LavaGapS7-v0")
env = FlattenObservationWrapper(env)
env = LogWrapper(env)
key = jax.random.PRNGKey(30)
key, _key = jax.random.split(key)
obs, state = env.reset(_key, None)

def render(obs, title):
    plt.imshow(obs)
    plt.title(title)
    plt.axis('off')
    plt.savefig("image.png")

render(observations.rgb(state.env_state.state), "Initial observation")
network = ActorCritic(env.action_space(None).n, activation=config["ACTIVATION"])
params = out['runner_state'][0].params

for t in range(100):
    rng, _rng = jax.random.split(rng)
    pi, _ = network.apply(params, obs)
    action = pi.sample(seed=_rng)
    print(action)
    obs, state, reward, done, info = env.step(rng, state, action, None)
    render(observations.rgb(state.env_state.state), f"Step {t}")
    input('Continue...')
```

The initial observation:
![initial_obs](https://github.com/epignatelli/navix/assets/34233348/b9ce675f-e313-4388-a4c8-af74338912e3)

After 2 steps:
![t2](https://github.com/epignatelli/navix/assets/34233348/db92e443-afd6-45eb-ba92-6c96079fe21b)

After 3 steps:
![t3](https://github.com/epignatelli/navix/assets/34233348/8faeb066-6f43-4c77-a109-911d5c9dfb62)

I had the same with the DoorKey environments, where agents simply walk through the dividing wall. Any ideas of what is happening?

Thanks!

Daniel

**Comment by epignatelli** [comment]
Hey! It's definitely a bug introduced by some new commit -- let me have a look into into today and I'll get back to you asap.

**Comment by epignatelli** [comment]
I just had a look, and spotted a bug.
It should be fixed in #84, let me know if that works for you.

You can install the dev branch with:
```sh
pip install git+https://github.com/epignatelli/navix.git@fix-walkable
```
